src/views/berichten/BerichtenList.vue

```diff
@@ -30,7 +30,7 @@ import { navigationStore, berichtStore } from '../../store/store.js'
 					</NcActionButton>
 				</NcActions>
 			</div>
-			<div v-if="berichtStore.berichtenList">
+			<div v-if="!!berichtStore.berichtenList?.length">

```

Comment:
`!!` is unnecessary as `?.length` will return either a number or `undefined`, both of these automatically count as booleans.

![image](https://github.com/user-attachments/assets/178b2633-1912-483c-acc3-98c5b4f23941)
![image](https://github.com/user-attachments/assets/e2ea3f61-1f01-4cca-a749-6bc4a5db75d4)

it does not need to be removed as this is not breaking and just a tiny thing.

Reply:
`!!` is optimally used when you want to get the boolean version of a specific value.